### PR TITLE
Enumarable PR

### PR DESCRIPTION
**Pull request by Kossi-stack**
In this pull request, these are the following requirements:
 - Create a class MyList that has an instance variable @list
 - Create a module MyEnumerable that implements the following methods:

       all?
       any?
       filter

### CODE REVIEW

File: mylist.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class MyList
```
Comment (henatan99) [inline mylist.rb:3]: - [x] Kindly import the `MyEnumerable` module using `require_relative`. Now it is raising an error.

Screenshot from 2021-11-12 21-18-10

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Kossi-stack , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Kindly rebase the base branch for your Pull Request to `main`. Always make sure you follow the correct GitHubFlow/GitFlow as it is a preliminary general requirement. :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mylist.rb (hunk 1 above)
Comment (elmar8287) [reply mylist.rb:3]: Fixed 💯

File: mylist.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require_relative 'my_enumerables'
```
Comment (rloterh) [inline mylist.rb]: This aspect requires further improvement:
- [x] File import is not pointing to the correct file, and therefore the app cannot start correctly. There is no file in your directory called _my_enumerable._ Kindly ensure to improve upon this aspect

**Error with file import:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kossi & Elmar, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Kindly take your time to correctly address those issues outlined in the previous code reviews, and please check carefully to ensure that the changes are working as intended._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mylist.rb (hunk 2 above)
Comment (elmar8287) [reply mylist.rb]: Fixed

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Thanks for the review!

**Pull request merged**